notebooks/WET/WET-007/WET-007_20241028_tie-vs-danish.ipynb

```diff
@@ -0,0 +1,344 @@
+{
```

Comment:
<p>It may help to add <code>comcam</code> or <code>observed</code> to the notebook title   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/ts_aos_analysis/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
It's a good point that at some point we will have all these notebook for LSSTCam too. Do you think we should append camera names like this, or create separate directories?

Reply:
putting `comcam` at the end would help imho,  we don't know if we'll be changing directory structure,  but if we do, it'll be easy to move them around.

Reply:
I talked to guillem about this, and he said he'd like to create subdirectories `notebooks/comcam`, etc. I will do this after these PRs are merged.